src/daos/build/install-single-host-dgw.sh

```diff
@@ -0,0 +1,272 @@
+#!/bin/bash
+
+# fail on errors
+set -e
+set -x
+
+function set_boolean()
```

Comment:
This was moved to a file

Reply:
you're right.  However in this specific case downloading it from the branch would be needed before setting the options for the repo & branch.  I did think about this one.  I'll change this one after the squash so it exists in the base branch before it gets used.